docs/api_reference/contacts_api/get_contact_by_phone.md

```diff
@@ -0,0 +1,50 @@
+---
+title: GET /contacts/phone/:phone
+sidebar_position: 4
+---
+
+# GET /contacts/phone/:phone
+
+Get a specific contact given a `phone_number`.
+
+### Required Parameters
+
+| Parameter         | Type   | Description                     |
+| :-----------------| :----- | :------------------------------ |
+| `phone_number`    | string | The phone number of the contact |
+
+### Example Request
+
+```bash title=request.sh
+curl -X GET "https://api.callbell.eu/v1/contacts/phone/579-037-2023" \
```

Comment:
In this example, rven if we sanitize the phone number, maybe we could just pass it without dashes to avoid confusion? WDYT?

```suggestion
curl -X GET "https://api.callbell.eu/v1/contacts/phone/5790372023" \
```